cmd/dagger/module.go

```diff
@@ -70,7 +70,9 @@ func init() {
 	moduleInitCmd.Flags().StringVar(&licenseID, "license", defaultLicense, "License identifier to generate - see https://spdx.org/licenses/")
 
 	modulePublishCmd.Flags().BoolVarP(&force, "force", "f", false, "Force publish even if the git repository is not clean")
-	modulePublishCmd.Flags().AddFlagSet(moduleFlags)
+	modFlag := *moduleFlags.Lookup("mod")
+	modFlag.Usage = modFlag.Usage[:strings.Index(modFlag.Usage, "containing that file.")-1]
```

Comment:
```suggestion
	modFlag.Usage = modFlag.Usage[:strings.Index(modFlag.Usage, " Either local path")-1]
```

This will produce desired output of
![image](https://github.com/dagger/dagger/assets/3187222/68191d82-6d98-4b5c-b345-4276ccb8149d)

Reply:
- space in " Either..." is intentional.
- produces consistent help text with no ending "." full-stop/period character.